src/pages/groupComparison/groupSelector/GroupSelectorButton.tsx

```diff
@@ -166,8 +171,10 @@ class GroupSelectorButton extends React.Component<
                             placement="top"
                             show={this.hovered}
                             target={this.button}
+                            onHide={this.onHideOverlay}
```

Comment:
curious about this one

Reply:
@alisman This fixes "Warning: Failed prop type: The prop onHide is marked as required in Overlay, but its value is undefined".

Reply:
this seems risky to me since there is is nothing functionally broken so far as i know.  if we're just trying to get rid of a warning, then maybe a noop?